services/movie_session.py

```diff
@@ -42,3 +42,17 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict] | str:
+    if movie_session_id:
+        movie_session = MovieSession.objects.get(id=movie_session_id)
```

Comment:
The `get_taken_seats` function does not handle the case where the `movie_session_id` does not correspond to an existing `MovieSession`. Consider handling the `DoesNotExist` exception to provide a more user-friendly error message.